src/users/src/users-service/handlers.go

```diff
@@ -122,7 +122,47 @@ func UserShowByIdentityId(w http.ResponseWriter, r *http.Request) {
 		panic(err)
 	}
 }
+// ClaimUser handler
+func ClaimUser(w http.ResponseWriter, r *http.Request) {
+	
+	enableCors(&w)
+	if (*r).Method == "OPTIONS" {
+		w.WriteHeader(http.StatusOK)
+		return
+	}
+	r.ParseForm()
+	identityId := r.Form.Get("identityId")
+	userId, err := strconv.Atoi(r.Form.Get("userId"))
```

Comment:
Sending in payloads using `application/x-www-form-urlencoded` content type breaks precedence across the other APIs where we're using JSON payloads. I think we should stay consistent or ideally, in this specific case, be more RESTful and use `POST /users/id/{userID}/claim`. I realize this starts to get into subjective territory on RESTful API design but it feels like putting the `claim` verb on the end of the exiting `/users/id/{userID}` is more intuitive.

Reply:
Makes sense. Also, I just realzied that we dont actually need the identityID to be passed since we just want to keep track on whether the user is claimed or not. Updated to use PUT /users/id/{userID}/claim